superset-frontend/src/filters/components/Select/SelectFilterPlugin.tsx

```diff
@@ -275,14 +292,15 @@ export default function PluginFilterSelect(props: PluginFilterSelectProps) {
         onSearch={searchWrapper}
         onSelect={clearSuggestionSearch}
         onBlur={handleBlur}
-        onDropdownVisibleChange={setIsDropdownVisible}
+        onDropdownVisibleChange={onDropdownVisibleChange}
```

Comment:
I think we can use a different (maybe a bit simpler?) approach here
```suggestion
        dropdownRender={originNode => {
          if (isDropdownVisible && !wasDropdownVisible) {
            originNode.ref?.current?.scrollTo(0);
          }
          return originNode;
        }}
```
and before that:
```
const wasDropdownVisible = usePrevious(isDropdownVisible);
```

What do you think?

Reply:
I think it's a really great solution. Much more elegant! I'll make the changes and test it. Thank you!